src/index.ts

```diff
@@ -318,7 +325,7 @@ export function startHeapProfiling(): void {
     const profile = pprof.heap.profile(undefined, config.sm)
     log('Heap profile collected...')
     uploadProfile(profile).then(() => log('Heap profile uploaded...'))
-  }, INTERVAL)
+  }, SAMPLE_RATE*10) // converting to milliseconds as setInterval expects milliseconds.
```

Comment:
It should be `SAMPLING_DURATION`

Reply:
Sorry but can you please explain? not sure if I am right, but it means for the `setInterval` function right? _(I thought it is for intervals between samples)_

Does it mean that for the time, till it will collect the sample?

Reply:
The latter, I think – moreover, in our case it also means how often the profiler uploads data to the server